ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/SubmissionValidator.scala

```diff
@@ -303,16 +318,20 @@ object SubmissionValidator {
 
   private lazy val engine = Engine()
 
+  val DefaultMaximumStateValueCacheSize: Long = 64L * 1024L * 1024L
```

Comment:
You can probably spare a couple of `L`s here

Reply:
Tried that; WartRemover hated it. 😞